speechbrain/lobes/augment.py

```diff
@@ -168,6 +172,8 @@ def mask_along_axis(self, x, dim):
             D = time
             n_mask = self.n_time_mask
             width_range = self.time_mask_width
+            if not self.time_mask_ratio == 0:
+                width_range = (0, int(self.time_mask_ratio * time))
```

Comment:
is it possible to have a negative value for `time_mask_ratio` ? If not, make sure to add an assert somewhere..